core/src/subgraph/instance_manager.rs

```diff
@@ -480,69 +479,69 @@ where
 
         // Process events from the stream as long as no restart is needed
         loop {
-            let block = match block_stream.next().await {
-                Some(Ok(BlockStreamEvent::ProcessBlock(block))) => block,
-                Some(Ok(BlockStreamEvent::Revert(subgraph_ptr))) => {
-                    info!(
-                        logger,
-                        "Reverting block to get back to main chain";
-                        "block_number" => format!("{}", subgraph_ptr.number),
-                        "block_hash" => format!("{}", subgraph_ptr.hash)
-                    );
+            let (block, cursor) = match block_stream.next().await {
+                Some(res) => match res {
```

Comment:
I see there was a change from `Some(Ok(...))` to `Some(res) => match res {`. Regardless of the style preference, mixing that into this PR makes the diff harder to read, so I'd prefer if we kept the previous style to minimize the diff.